aiobotocore/endpoint.py

```diff
@@ -259,23 +219,11 @@ def __init__(self, host,
             conn_timeout=self._conn_timeout,
             skip_auto_headers={'CONTENT-TYPE'},
             response_class=ClientResponseProxy,
-            loop=self._loop)
+            loop=self._loop,
+            auto_decompress=False)
 
     @asyncio.coroutine
     def _request(self, method, url, headers, data):
-        # Note: When using aiobotocore with dynamodb, requests fail on crc32
-        # checksum computation as soon as the response data reaches ~5KB.
-        # When AWS response is gzip compressed:
-        # 1. aiohttp is automatically decompressing the data
-        # (http://aiohttp.readthedocs.io/en/stable/client.html#binary-response-content)
-        # 2. botocore computes crc32 on the uncompressed data bytes and fails
-        # cause crc32 has been computed on the compressed data
-        # The following line forces aws not to use gzip compression,
-        # if there is a way to configure aiohttp not to perform decompression,
-        # we can remove the following line and take advantage of
-        # aws gzip compression.
-        # See: https://github.com/aio-libs/aiohttp/issues/1992
-        headers['Accept-Encoding'] = 'identity'
```

Comment:
@jettify bug:  we need to keep this identity until https://github.com/boto/botocore/issues/1255 is fixed

Reply:
Got it, will bring back this lines.